app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -2,7 +2,7 @@
   <div
     class="conversation"
     :class="{ active: isActiveChat, 'unread-chat': hasUnread }"
-    @click="cardClick(chat)"
```

Comment:
Hey @wwwbruno since we are no longer using the refs, we can omit changes in this file.

Reply:
https://github.com/chatwoot/chatwoot/pull/757/commits/db5c3e4916baa0f680bfcf0050902adbc773fffb